modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -50,16 +49,24 @@ spec:
       version: v1
       resource: deployments
 ----
-
-A service binding request is created and the {servicebinding-title} controller projects the database service connection information into the application deployment as files by using a volume mount. After the request is successful, the application is redeployed and the connection is established.
-
++
+A service binding request is created and if the database service connection request is successful, the application is redeployed and the connection is established.
++
 .Binding connector
 image::odc-binding-connector.png[]
-
++
 [NOTE]
 ====
 You can also use the context menu by dragging the dangling arrow to add and create a binding connection to an operator-backed service.
 
 .Context menu to create binding connection
 image::odc_context_operator.png[]
 ====
+
+. In the navigation menu, click *Topology*. The spring-petclinic deployment in the Topology view includes an Open URL link to view its web page.
+
+. Click the *Open URL* link.
```

Comment:
I wonder if it makes sense to combine these two steps into one.

I mean something like this:

```suggestion
. In the navigation menu, select *Topology* -> *Open URL*.
```

Also, as a reader, I find the following sentence redundant because it's the next step:
_The spring-petclinic deployment in the Topology view includes an Open URL link to view its web page._

Reply:
The Open URL link is not very evident in the UI. Hence the sentence is included there.

The new suggestion does not state clearly where to find the Open URL link in the Topology view.

Reply:
Keeping it as is.